### PR TITLE
ManageIQ User Survey Blog Post

### PR DESCRIPTION
**Pull request by terezanovotna**
User Survey post @cybette

### CODE REVIEW

**Comment by serenamarie125** [comment]
Looks good @terezanovotna, thanks for doing this 👏

File: site/_posts/2018-02-26-manageiq-survey.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Help Us Improve!
```
Comment (cybette) [inline site/_posts/2018-02-26-manageiq-survey.md]: Might be good to be a bit more specific in the title, perhaps "Quad Icons: Help Us Improve!"

File: site/_posts/2018-02-26-manageiq-survey.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+date: 2018-02-26 09:05 EST
```
Comment (cybette) [inline site/_posts/2018-02-26-manageiq-survey.md]: If you make the title change and then date change to today (Feb 28), I can merge this. Thanks!

**Comment by terezanovotna** [comment]
ready to be merged with March 1st date @cybette @serenamarie125

**Comment by cybette** [comment]
@terezanovotna now there are 2 files in the PR, which will result in 2 blog posts. please delete 2018-02-26-manageiq-survey.md , thanks!

**Comment by terezanovotna** [comment]
@cybette Thanks! 👏👏👏

**Pull request merged**